search_gov_spiders/README.md

```diff
@@ -0,0 +1,85 @@
+# Search_gov_spiders Info
```

Comment:
Can we further flatten this?  Everything in the current top-level `search_gov_spiders` goes to the repo root, then we just have a single-depth `search_gov_spiders` (which itself contains `spiders`) underneath that?  I just have a sense that `search_gov_spiders/search_gov_spiders` is going to get unnecessarily confusing super quickly.  For example, we'd no longer have two different READMEs to keep track of and keep up to date; `requirements.txt` wouldn't be hidden in a subdirectory; etc.